.github/workflows/generate-outputs.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will install Python dependencies, run the scripts to generate output and commit it back to the same branch
+
+name: Berean-Build
+
+on:
+  push:
```

Comment:
I think restricting this to only run when the files inside the `input` directory are changed be better: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files

Reply:
changed :+1: